src/components/Footer/Footer.tsx

```
@@ -39,35 +39,67 @@ const Footer = () => {
               </ul>
             </div>
             <nav className="buttons">
-              <a className="btn-footer" type="button" title="Instagram" target='_blank' href="https://www.instagram.com" aria-label='Instagram'>
+              <a
+                className="btn-footer"
+                type="button"
+                title="Instagram"
+                target="_blank"
+                href="https://www.instagram.com"
+                aria-label="Instagram"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)